docs/API.md

```diff
@@ -2057,21 +2057,16 @@ Request body:
 
 #### GET /macaroon
 
-This endpoint returns a macaroon in JSON format that, when its third
-party caveats are discharged, will allow access to the charm store. No
+This endpoint returns a macaroon in JSON format that, when its third party caveats are discharged, will allow access to the charm store. No
 prior authorization is required.
 
 #### GET /delegatable-macaroon
 
 This endpoint returns a macaroon in JSON format that can be passed to
 third parties to allow them to access the charm store on the user's
-behalf.  A first party "is-entity" caveat may be added to restrict those
-parties so that they can only access a given charmstore entity with a
-specified id.
+behalf. The entity id must be added as a url encoded parameter and is added as a first party "is-entity" caveat to restrict the usability of the returned macaroon, so that third parties can only access a given charmstore entity with a the specified id.
```

Comment:
This doesn't say what the name of the URL-encoded parameter is.
Let's document the URL parameters similarly to other examples in this
doc (e.g. in GET meta/endpoint).

Also, I'm not sure we need to discuss the exact caveats that are added here - that's
really an implementation detail (they are *first*-party caveats after all).

Perhaps simply something like:

"If the id parameter is specified, the returned macaroon will be restricted for
use only with the entity with the given id."

?